tests/unit/Common/Internal/Authentication/StorageAuthSchemeTest.php

```diff
@@ -94,7 +95,13 @@ public function testComputeCanonicalizedResourceMockMultipleValues()
     {
         $queryVariables = array();
         $queryVariables['COMP'] = 'list';
-        $queryVariables[Resources::QP_INCLUDE] = 'snapshots,metadata,uncommittedblobs';
+        $queryVariables[Resources::QP_INCLUDE] = ServiceRestProxy::groupQueryValues(
```

Comment:
@katmsft This should be ordering the values for the following call to `computeCanonicalizedResourceMock`.